web/models/stripe_connect_account.ex

```diff
@@ -6,28 +6,67 @@ defmodule CodeCorps.StripeConnectAccount do
   use CodeCorps.Web, :model
 
   schema "stripe_connect_accounts" do
-    field :business_name, :string
+    # these might not be needed by managed accounts,
+    # but we have them defined at DB level
+
     field :business_url, :string
-    field :charges_enabled, :boolean
-    field :country, :string
     field :default_currency, :string
     field :details_submitted, :boolean
     field :display_name, :string
-    field :email, :string
-    field :id_from_stripe, :string, null: false
-    field :managed, :boolean
     field :support_email, :string
     field :support_phone, :string
     field :support_url, :string
-    field :transfers_enabled, :boolean
 
+    # this was needed for connect
     field :access_code, :string, virtual: true
 
+    # these fields are filled out and sent by the client
+
+    field :address1, :string, virtual: true
+    field :address2, :string, virtual: true
+    field :business_ein, :string, virtual: true
+    field :business_name, :string
+    field :business_type, :string, virtual: true
+    field :city, :string, virtual: true
+    field :country, :string
+    field :dob_day, :integer, virtual: true
+    field :dob_month, :integer, virtual: true
+    field :dob_year, :integer, virtual: true
+    field :email, :string
+    field :first_name, :string, virtual: true
+    field :last_name, :string, virtual: true
+    field :recipient_type, :string, virtual: true
+    field :ssn_last4, :string, virtual: true
+    field :state, :string, virtual: true
+    field :zip, :string, virtual: true
+
     belongs_to :organization, CodeCorps.Organization
 
+    # these fields are set automatically
+    field :managed, :boolean
+
+    # these fields are set from stripe
+    field :id_from_stripe, :string, null: false
+    field :charges_enabled, :boolean
+    field :transfers_enabled, :boolean
     timestamps()
   end
 
+  @managed_create_fields [
+    :address1, :address2, :business_ein, :business_name, :business_type,
+    :city, :country, :dob_day, :dob_month, :dob_year, :email,
+    :first_name, :last_name, :recipient_type, :ssn_last4,
+    :state, :zip, :organization_id,
+    :managed, :id_from_stripe, :charges_enabled, :transfers_enabled
+  ]
+
+  def managed_create_changeset(struct, params) do
```

Comment:
Don't think we want distinction between `create` and `managed_create`. Also, `create` fields will be just `country` and  `managed`, where the latter is always `true`.